packages/lit-dev-content/site/tutorials/content/svg-templates/03.md

```diff
@@ -0,0 +1,102 @@
+### Overview
+
+This is chapter of the demo covers the basics of `<clipPath>` and
+`<g>` in SVG including how to:
+
+- create a clip path
+- create a group of groups with `<use>`
+- apply a clip path to a group
+
+### Learn
+
+In pattern making, a _tile_ is a polygon that can be repeated across
+a two dimensional plane. The tile in this demo will be a series of
+transformed texts.
+
+Clip paths can be used to gather a group of visual elements. In SVG,
+a clip path is defined with the `<clipPath>` element.
+
+In the example below, a clip path will display contents within a 
+rectangle with an id of `rect-clip`.
+
+```TS
+const helloClipPath = svg`
+	<clipPath id="rect-clip">
+		<rect width="200" height="200"></rect>
+	</clipPath>
+`;
+```
+
+In the example below a 300px black square will be clipped by the 
+clip-path above. As a result, tjhe 300px square visually appears to be
+a 200px black square.
+
+```TS
+const helloTile = svg`
+	<rect
+		clip-path="url(#rect-clip)"
+		width="300"
+		height="300"
+		fill="#000">
+	</rect>
+```
+
+### Apply
+
+This demo uses a `<clipPath>` to create a tile containing five groups
+of rotated text.
+
+To accomplish this, first, create a clip path composed of a 200px
+square.
+
+```TS
+const createClipPath = () => svg`
+	<clipPath id="rect-clip">
+		<rect width="200" height="200"></rect>
+	</clipPath>
+`;
+```
+
+After, move the _motif_ inside `<defs>` and give it an id called
+`chars-rotated`.
+
+```TS
+@customElement('repeat-pattern')
+export class RepeatPattern extends LitElement {
+
+	...
+
+	render() {
+		return html`
+			<svg>
+				<defs>
+					${createClipPath()}
+					${createElement(this.chars)}
+					${createMotif(
+						this.numPrints,
+						this.rotationOffset,
+					)}
+				</defs>
+			</svg>
+    	`;
+	}
+}
+```
+
+Now that the motif is available in `<defs>`, make a function called
+`createTile` that returns a group of rotated texts via `<use>`.
+
+The placement and count is arbitrary. Try different variations! The
+demo uses five motifs to create an offset clipped along the x axis.
+
+```TS
```

Comment:
Could this leverage lit templating to make it easy to edit one spot to generate many?

Reply:
i think so, this spot is where the "tiles" are created. So anything done here is the "repeat" part of the pattern